.github/workflows/smoke.yml

```diff
@@ -105,8 +110,17 @@ jobs:
         path: .tmp/smokehouse-ci-failures/
 
   smoke-fr:
+    strategy:
+      matrix:
+        smoke-test-shard: [1, 2, 3]
```

Comment:
Three shards for all of these is getting a bit excessive (that list gets realllllly long) since we're not bottle necked on them. Should we use 2 for everything? After #13943 they should still be quite snappy

Reply:
only reason I did 3 was to slightly help the "re run failures because flaky test" use case.

although... maybe we could write to the job cache what smokes failed...then only re-run the failures on retries... same problem, better solution than "split it up more"